letsencrypt/acme/other.py

```diff
@@ -55,7 +100,7 @@ def from_msg(cls, msg, key, nonce=None):
         logging.debug('%s signed as %s', msg_with_nonce, sig)
 
         return cls(alg='RS256', sig=sig, nonce=nonce,
```

Comment:
Somewhat hinted in https://github.com/letsencrypt/lets-encrypt-preview/pull/267#discussion_r25548605, I've already implemented all signature schemes from JWA and added necessary "dispatch" logic in my local branch :)